### ISSUE TITLE
Fuzzy finder "Show more" button and search link are not accessible due to tab key override

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#44124]
The fuzzy finder overrides the <kbd>tab</kbd> key to switch between symbol types and has some controls that don't have keyboard shortcuts, like the "Show more" button and the search link at the bottom.

An easy fix would be to remove the <kbd>tab</kbd> override, since all the symbol types now have independent keyboard shortcuts.
